web/impact/impact/v1/helpers/mentor_program_office_hour_helper.py

```diff
@@ -0,0 +1,61 @@
+# MIT License
+# Copyright (c) 2019 MassChallenge, Inc.
+
+from accelerator.models import MentorProgramOfficeHour
+from impact.v1.helpers.model_helper import (
+    ModelHelper,
+    PK_FIELD,
+    OPTIONAL_INTEGER_FIELD,
+    OPTIONAL_STRING_FIELD,
+)
+
+OFFFICE_HOUR_FIELDS = {
+    "id": PK_FIELD,
+    "title": OPTIONAL_STRING_FIELD,
+    "location": OPTIONAL_STRING_FIELD,
+    "start_time": OPTIONAL_STRING_FIELD,
+    "end_time": OPTIONAL_STRING_FIELD,
+    "date": OPTIONAL_STRING_FIELD,
+    "mentor_id": OPTIONAL_INTEGER_FIELD,
+    "mentor_name": OPTIONAL_STRING_FIELD,
+    "mentor_email": OPTIONAL_STRING_FIELD,
+    "finalist_id": OPTIONAL_INTEGER_FIELD,
+    "finalist_name": OPTIONAL_STRING_FIELD,
+    "finalist_email": OPTIONAL_STRING_FIELD,
+}
```

Comment:
I believe the `reservation status` is missing in this list..it's specified on the ticket